### ISSUE TITLE
Support home/end keys in amp-selector

### ISSUE THREAD

**Issue by sparhami** [ampproject/amphtml#23905]
> You can do something like the following jsbin: https://jsbin.com/ledopojawe/edit?html,output

> This uses `<amp-selector>` with `keyboard-select-mode="focus"` to move focus over as you navigate with arrow keys. It does not have support for the home/end keys, which is something we can look into adding.

_Originally posted by @sparhami in https://github.com/ampproject/amphtml/issues/23879#issuecomment-520589015_

**Comment by leafsy** [comment]
I'd like to pick up this issue; my assumption is that home key would bring focus to the first element in the selector (leftmost one if document is LTR) and end key to the last element?

**Comment by sparhami** [comment]
> I'd like to pick up this issue; my assumption is that home key would bring focus to the first element in the selector (leftmost one if document is LTR) and end key to the last element?

Yes, I think that is all that needs to be done here.
